#name of commands

command_num: 5   #remember that is enumerated from 0 you have to set it manually 
command:
  command0:
    name: "bailar" #if it's set to 0,0,0 robot will go back to start point 
    x:    0.0
    y:    0.0
    angle: 0.0

  command1:
    name: "vuelta"
    x:    0.0
    y:    0.0
    angle: 0.0

  command2:
    name: "la patita"
    x:    0.0
    y:    0.0
    angle: 0.0   

  command3:
    name: "traer chocolate"
    x:    0.874
    y:    1.96
    angle: 45.0
    
  command4:
    name: "llevar pintura"
    x:    3.252
    y:    0.039
    angle: 236.0

